tasmota/i18n.h

```diff
@@ -211,6 +211,8 @@
 #define D_JSON_IMPORT "Import"
 #define D_JSON_EXPORT "Export"
 #define D_JSON_TOTAL_ACTIVE "TotalActive"
+#define D_JSON_SIGNALSTRENGTH "Signal Strength"
+#define D_JSON_CHIPTEMPERATURE "Chip Temperature"
```

Comment:
Remove spaces from JSON field names (Tasmota Standard)

Reply:
Thanks for your remark and changed to be Tasmota Standard compliant.